File: src/productos/Limpieza.java
package productos;

public class Limpieza extends Producto {
    private String aroma;


    public Limpieza(String nombre, float precio, String fechaImportacion, int numeroSerie, int stock, String aroma) {
        super(nombre, precio, fechaImportacion, numeroSerie, stock);
        this.aroma = aroma;
    }

    public String getAroma() {
        return aroma;
    }
}

File: src/productos/Limpieza.py
from productos.Producto import Producto


class Limpieza(Producto):
    def __init__(self, nombre, precio, fecha_importacion, numero_serie, stock, aroma):
        super().__init__(nombre, precio, fecha_importacion, numero_serie, stock)
        self.__aroma = aroma

    def get_aroma(self):
        return self.__aroma
